packages/sources/peregrine-fund-admin/src/config/index.ts

```diff
@@ -0,0 +1,31 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const DEFAULT_BASE_URL_VALUE =
+  'https://fund-admin-data-adapter-v1-960005989691.europe-west2.run.app'
+export const DEFAULT_NAV_URL_VALUE = '/api/v1/nav/'
+export const DEFAULT_RESERVE_URL_VALUE = '/api/v1/reserve/'
```

Comment:
Do not declare variables for these. We usually treat this file as config not code

Reply:
Removed and refactored